lit_gpt/adapter_v2.py

```diff
@@ -84,8 +84,6 @@ def from_name(cls, name: str, **kwargs: Any) -> Self:
     def _init_weights(self, module: nn.Module) -> None:
         """Meant to be used with `gpt.apply(gpt._init_weights)`. Unused method left for completeness."""
         super()._init_weights(module)
-        if isinstance(module, CausalSelfAttention):
-            module.reset_parameters()
```

Comment:
This is done through super(), because this module's CSA subclasses the adapter CSA